src/app/components/layout/layout.component.css

```diff
@@ -1,11 +1,28 @@
 main {
+  padding-top: 2rem;
   background-color: var(--white);
   padding: 1.6rem;
   display: grid;
   grid-template-columns: 30rem 1fr;
   gap: 3rem;
-  min-width: 94rem;
+  max-width: 85%;
+  margin-inline: auto;
+  max-height: 100dvh;
   border-radius: 1rem;
   min-height: 60rem;
   box-shadow: #64646f33 0 0.7rem 2.9rem;
 }
+
+
+@media screen and (min-width: 1280px) {
```

Comment:
use defined breakpoint values instead of hard coding the screen widths

Reply:
Please the breakpoint is not hard coded, got it from tailwind  and blog.logrocket.com

https://blog.logrocket.com/css-breakpoints-responsive-design/

https://tailwindcss.com/docs/screens